ci/cpu/asan_ubsan_lsan.yml

```diff
@@ -17,9 +17,7 @@ cpu asan ubsan lsan build:
   needs:
     - cpu asan ubsan lsan deps
   variables:
-    DEPLOY_IMAGE: $CSCS_REGISTRY_PATH/cpu-asan-ubsan-lsan/deploy:$CI_COMMIT_SHA
-    # For symbolizing stacktraces with llvm-symbolizer
-    EXTRA_APTGET_DEPLOY: "llvm-18"
```

Comment:
I suppose `llvm-18` is pulled by `clang-18` and no longer needed explicitly?

Reply:
We are not dealing with a clean image anymore. All the apt packages are installed in the `build deps` phase and clang-18 depends on llvm-18.